src/utils/optimizely.js

```diff
@@ -0,0 +1,65 @@
+export const EVENTS = {
+  ENROLLMENT_CLICK: 'enterprise_learner_portal_enrollment_click',
+}
+
+export const EXPERIMENT_0 = {
+  ID: 21468970242,
+  VARIANT_IDS: {
+    CONTROL: 21446530045,
+    VARIANT_1: 21458980337,
+  },
+}
+
+export const getOptimizelyProps = () => ({
+  src: process.env.ENVIRONMENT === 'production'
+    ? `${process.env.BASE_URL}/optimizelyjs/${process.env.OPTIMIZELY_PROJECT_ID}.js`
+    : `https://cdn.optimizely.com/js/${process.env.OPTIMIZELY_PROJECT_ID}.js`,
+  rel: 'preload',
+});
+
+export const getActiveExperiments = () => {
+  window['optimizely'] = window['optimizely'] || [];
```

Comment:
nit: Is there a reason to use `window['optimizely']` vs. `window.optimizely`? Should these be consistent?

Reply:
That's just copypasta from two different examples in documentation.